tasks/main.yml

```diff
@@ -7,7 +7,40 @@
     name: "{{ item }}"
   with_items:
     - epel-release
+
+- name: Ensure Gluster 3.x Centos repository is absent
+  package:
+    name: "{{ item }}"
+    state: absent
+  become: yes
+  with_items:
     - centos-release-gluster
+
+- name: Gluster-4.1 repo
+  yum_repository:
+    name: centos-gluster-4.1
+    description: CentOS Storage SIG repo for gluster 4.1
+    baseurl: http://mirror.centos.org/centos/$releasever/storage/$basearch/gluster-4.1
+    gpgkey: https://raw.githubusercontent.com/CentOS-Storage-SIG/centos-release-storage-common/master/RPM-GPG-KEY-CentOS-SIG-Storage
+    enabled: yes
+  become: yes
+
+- name: Ceph Luminous
```

Comment:
I don't think this is required for Gluster? I assume there is a Ceph role floating around that this should be in?

Reply:
Oops, looks like its probably there by mistake!